src/main/resources/assets/betterquesting/lang/en_US.lang

```diff
@@ -71,6 +71,8 @@ betterquesting.gui.remaining_lives=Remaining Lives: %s
 betterquesting.gui.full_lives=Lives Full
 betterquesting.gui.closing_warning=Are you sure you want to close this editor?
 betterquesting.gui.closing_confirm=Changes may be lost!
+betterquesting.gui.claim_all_warning=Are you sure you want to claim all rewards?
+betterquesting.gui.claim_all_confirm=There can be a lot of items here that will drop to the ground if the inventory is full!
```

Comment:
> There can be a lot of items here that will drop to the ground if the inventory is full!

There may be a lot of items that will drop to the ground if your inventory is full!